### ISSUE TITLE
Circle CI broken?

### ISSUE THREAD

**Issue by flanfly** [randombit/botan#498]
The URL the script tries to download disappeared.

```
wget -q -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -

gpg: no valid OpenPGP data found.

wget -q -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add - returned exit code 2

Action failed: wget -q -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
```

**Comment by randombit** [comment]
http://llvm.org/apt/ says "APT mirror was temporary switched off due to excess load. We are working on bringing it back. Stay tuned!"

Maybe there is a better way for our CircleCI setup to handle this than downloading a fresh copy of clang with every build... unfortunately Ubuntu 12 included Clang is back at 3.0, which is (IIRC) missing features we need.

**Comment by randombit** [comment]
Fixed now
